target/linux/ipq806x/image/Makefile

```diff
@@ -244,6 +244,18 @@ define Device/qcom_ipq8064-db149
 endef
 TARGET_DEVICES += qcom_ipq8064-db149
 
+define Device/qxwlan_e5200
+	$(call Device/LegacyImage)
+	$(call Device/UbiFit)
+	DEVICE_DTS := qcom-ipq8064-qxwlan-e5200
```

Comment:
Please have a look at recently merged commit e0382e88a919d342dd6fbf07971ceab15d0b6aaf.
Implications for you:
- Rename DTS files by removing "qxwlan-"
- Remove DEVICE_DTS here and add `SOC := qcom-ipq8064`
- Update 0069-arm-boot-add-dts-files.patch and DTS include lines.
- Same for to-be-added 4.19 changes.

Reply:
OK,
I have modified and I tested my device on the latest master.
Wait until e5200 is supported on 4.14 kernel I will add 4.19 kernel support on the latest master branch